### ISSUE TITLE
Is there any way to limit the input method as English US Keyboard in desktop app?

### ISSUE THREAD

**Issue by gscheartA** [microsoft/microsoft-ui-xaml#5044]
I am developing a app using WinUI3, and I would met some issue when I use other input method in some webview2 page, which lead to crash. Is there any way to limit the input method as English US Keyboard and could not switch in the run time of this app? Thank you for your help.

![image](https://user-images.githubusercontent.com/74002809/118777769-014b3300-b8bc-11eb-8553-480997039df2.png)

**Comment by StephenLPeters** [comment]
I don't think so, I think the keyboard is abstracted away from the app which doens't have knowledge of it.  @MikeHillberg or @Austin-Lamb is that right?

**Comment by mdtauk** [comment]
> I am developing a app using WinUI3, and I would met some issue when I use other input method in some webview2 page, which lead to crash. Is there any way to limit the input method as English US Keyboard and could not switch in the run time of this app? Thank you for your help.
> 
> ![image](https://user-images.githubusercontent.com/74002809/118777769-014b3300-b8bc-11eb-8553-480997039df2.png)

Got to wonder why you would want to restrict the keyboard input, and what would happen if the user does not have English US as one of their keyboard layout choices.  (For example I use en-gb)

**Comment by gscheartA** [comment]
Hi @mdtauk  and @StephenLPeters  . Thank you for your answer.
 I am sorry that I did not expressed clearly. This issue is that sometimes I input some content from keyboard and I sue some other input method， like this
![image](https://user-images.githubusercontent.com/74002809/119070759-3621cc00-ba1b-11eb-82d1-3de6c8102b02.png)
And then it will lead to crash for some unknown reason.
I would let user use their native input method such as English US Keyboard or en-gb...
Another way is let the app just receive the english letter or number from keyboard, I don't know if I could capture user input and directly switch to the letter?

**Comment by codendone** [comment]
Is it the WebView2 process which crashes? If so, and issue should be logged in the WebView2Feedback repo for that. Do you have specific repro steps to hit the crash?
